### PR TITLE
Add pre-commit to poetry

### PR DESCRIPTION
**Pull request by camilamaia**
`pre-commit` was not being installed. When I try to add it, I faced this error:

```shell
$ poetry add -D pre-commit
Using version ^2.6.0 for pre-commit

Updating dependencies
Resolving dependencies... (0.0s)

[SolverProblemError]
The current project's Python requirement (^3.6) is not compatible with some of the required packages Python requirement:
  - pre-commit requires Python >=3.6.1

Because no versions of pre-commit match >2.6.0,<3.0.0
 and pre-commit (2.6.0) requires Python >=3.6.1, pre-commit is forbidden.
So, because scanapi depends on pre-commit (^2.6.0), version solving failed.
```

for this reason, I increase the python version from 3.6 to 3.7 too

### CODE REVIEW

**Comment by camilamaia** [comment]
> @camilamaia your pull request is missing a changelog!

Since it is only a change for dev usage, I don't believe this is needed.

**Comment by supakeen** [comment]
The change from `3.6` to `3.7` is definitely a breaking change that should be noted in the CHANGELOG. It will make it so `pip` and friends won't find the scanapi package anymore on Python's below 3.7.

**Comment by vinigfer** [comment]
> Since it is only a change for dev ...

Realized you were arguing with a bot right? 😄 
Don't blame it. He was programmed to do that hahaha

File: pyproject.toml [hunk 1]
```diff
@@ -30,6 +30,7 @@ pytest-freezegun = "0.4.1"
 pytest-mock = "3.2.0"
 requests-mock = "1.8.0"
 black = "19.10b0"
+pre-commit = "2.6.0"
```
Comment (vinigfer) [inline pyproject.toml:33]: Just wondering: is poetry responsible for installing pre-commit hooks? If yes, then we might need to update here: https://github.com/scanapi/scanapi/blob/master/.pre-commit-config.yaml#L3
Made a PR on black project about 10 days ago. They no longer have the "stable" tag, which was causing my pre-commit install to fail.

**Comment by camilamaia** [comment]
> The change from `3.6` to `3.7` is definitely a breaking change that should be noted in the CHANGELOG. It will make it so `pip` and friends won't find the scanapi package anymore on Python's below 3.7.

@supakeen indeed, you are right. And specially because this is only a dev dependency. I made the change less dramatic then. Changed to ^3.6.1

**Comment by camilamaia** [comment]
> > Since it is only a change for dev ...
> 
> Realized this discussion was with a bot right? 😄
> Don't blame it. It was programmed to do that hahaha

@vinigfer yeah, I like to argue with bots, usually is easier than humans 😂 
I should program him to answer as well lol

File: pyproject.toml (hunk 1 above)
Comment (camilamaia) [reply pyproject.toml:33]: > Just wondering: is poetry responsible for installing pre-commit hooks? If yes, then we might need to update here: https://github.com/scanapi/scanapi/blob/master/.pre-commit-config.yaml#L3
> Made a PR on black project about 10 days ago. They no longer have the "stable" tag, which was causing my pre-commit install to fail.

@vinigfer Good catch! Even in their doc the `stable` is still there. Good to know. I updated it. Would you mink to take another look if it is all good?

Comment (vinigfer) [reply pyproject.toml:33]: Really? 🤔 
Hum ... the PR I made only changed README.md
Guess will check the docs later and open a new PR

I'll review it again in a couple of minutes ;)

**Pull request merged**